config/routes.rb

```diff
@@ -2,5 +2,8 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  get "/users/:id" => "users#show", as: 'user'
+  get "/users/:id/posts" => "posts#index"
+  get "/users/:id/posts/:id" => "posts#show", as: 'post'
```

Comment:
- [ ] [Optional] [Highly Recommended] It would be much better professionally if you add your routes like that:
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
end

# Defines the root path route ("/")
root "users#index"
```

Reply:
Thank you for the suggestion.